config/index.js

```diff
@@ -17,6 +17,9 @@ var conf = convict({
     format: [ 'dev', 'test', 'stage', 'prod' ],
     env: 'NODE_ENV'
   },
+  geodb: {
+    dbPath: path.resolve(__dirname, '../node_modules/fxa-geodb/db/cities-db.mmdb')
```

Comment:
You can also probably set the `env` value to something like `GEODB_DBPATH`. and for the kill switch (another config value under `geodb`) -> `GEODB_ENABLED` or something